torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -107,7 +107,7 @@ class IrParser {
 
       // Merge all dimensions because we're only supporting pointwise
       while (out->nDims() > 1)
-        out->merge(0);
+        out->merge(0, 1);
```

Comment:
this may not be needed - i see we have an overload with a single argument?

Reply:
I wanted to update it to be explicit as that's where Jie works, and I wanted him to see the syntax change :-)